test/ActionParameterHandler.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { ActionParameterHandler } from '../src/ActionParameterHandler';
+import { ActionMetadata, ControllerMetadata, ExpressDriver, ParamMetadata } from '../src';
+import { ActionMetadataArgs } from '../src/metadata/args/ActionMetadataArgs';
+import { ControllerMetadataArgs } from '../src/metadata/args/ControllerMetadataArgs';
+
+const expect = require('chakram').expect;
+
+describe('ActionParameterHandler', () => {
+  it('handle', async () => {
```

Comment:
You should add negative and positive cases

Reply:
Broader test coverage was added including two negative tests:
 - https://github.com/typestack/routing-controllers/pull/708/files#diff-5c248b2b7632d75cc5c2d989e5b9387fe961cb04f00ef500c93b55ccc93e1a15R128
 - https://github.com/typestack/routing-controllers/pull/708/files#diff-5c248b2b7632d75cc5c2d989e5b9387fe961cb04f00ef500c93b55ccc93e1a15R148